### ISSUE TITLE
Usage of ViewSetupHints is stated too strong

### ISSUE THREAD

**Issue by pbuts** [buildingSMART/BCF-XML#128]
Since pull request #127 the following is now stated in the documentation:

```
#### ViewSetupHints
 +This element contains information about the default visibility for elements of certain types (`SpacesVisible`, `SpaceBoundariesVisible` and `OpeningsVisible`) that should be applied if not stated otherwise.
```
This is too strong, the hints are just there to let the receiver of the issue know how the view was setup when the issue was created. 

Those settings may not influence the component sets at all, so it is up to the client displaying the viewpoint whether to use this info. 

=> 'should be applied if not stated otherwise' must be rephrased).

**Comment by GeorgDangl** [comment]
We'll remove the `should be applied if not stated otherwise` sentence since it's a bit vague and confusing.

**Comment by GeorgDangl** [comment]
This is already  fixed via efdb08e2560526c6545b2cade1c1292f75a8cfcf
